src/modules/common/components/SearchInput.js

```diff
@@ -11,19 +10,21 @@ export class SearchInput extends React.Component {
     filterText: PropTypes.string.isRequired,
     onFilterTextChange: PropTypes.func.isRequired,
     spaceSearch: PropTypes.bool,
+    onClick: PropTypes.func,
     t: PropTypes.func.isRequired
   }
 
   render () {
     return (
-      <div className={cx({ [style.searchSpacing]: this.props.spaceSearch })}>
+      <div className={this.props.spaceSearch ? style.searchSpacing : ''}>
```

Comment:
I prefere passing an object instead of an empty string :/

Reply:
empty object? as a classname?

Reply:
I mean the syntax I used before, but I'm also ok with the trinary